pom.xml

```diff
@@ -54,7 +54,10 @@ limitations under the License.
   </mailingLists>
 
   <properties>
-    <calciteVersion>1.22.0</calciteVersion>
+    <!-- Update this number when syncing with a new release of Apache Calcite -->
+    <calciteVersion>1.21.0</calciteVersion>
+    <!-- Update this to the commit hash of the final commit that produced the version above -->
+    <calciteCommitHash>3f7bbae47ad113121d3885ee70d2aa246490a033</calciteCommitHash>
```

Comment:
Dont really understand why we need that, but if it's required to make it work, it's OK to me.

Reply:
Needed to calculate the number of custom commits we have made on top of the last sync with Calcite. Currently we have synced with Calcite 1.21.0 and we have 30 commits on top so the version we generate is 130. If we commit once more, it will be 131 and so on

Suppose we sync with Calcite and now calcite is at 1.21.356, then if we update this commit hash we can start at 1.21.356.101 again. Its not strictly required -- if we hardcode some commit hash in the publishing script, that will work too but then we would have something like 1.21.356.132 (i.e. last number is ever increasing). Forgetting to update this also has no effect but I think its good hygiene